src/components/primitives/Links/ExternalLink.tsx

```diff
@@ -0,0 +1,26 @@
+import {} from 'components/ActionsBuilder/CallDetails/CallDetails.styled';
+import { FiExternalLink } from 'react-icons/fi';
+import { StyledSegmentLink, LinkDetail } from './styles';
+import { ExternalLinkProps } from './types';
+
+export const ExternalLink: React.FC<ExternalLinkProps> = ({
+  href,
+  children,
+  unstyled,
+}) => {
+  if (unstyled) {
+    return (
+      <StyledSegmentLink href={href} target="_blank" rel="noopener">
+        <LinkDetail>{children}</LinkDetail>
+        <FiExternalLink />
+      </StyledSegmentLink>
+    );
+  }
+
+  return (
+    <StyledSegmentLink href={href} target="_blank" rel="noopener">
+      <LinkDetail>{children}</LinkDetail>
+      <FiExternalLink />
+    </StyledSegmentLink>
+  );

```

Comment:
I think this should be only one component and handle the styling logic in the styled component

Reply:
thanks! yes i didn't realize this is not needed anymore.